packages/api-providers/src/abstracts/webb-provider.interface.ts

```diff
@@ -43,21 +43,21 @@ export interface WebbMixer<T extends WebbApiProvider<any>> {
   // deposit
   deposit: WebbMethod<MixerDeposit<T, DepositPayload>, MixerDepositEvents>;
   // withdraw
-  withdraw: WebbMethod<MixerWithdraw<T>, MixerWithdrawEvents>;
+  withdraw: WebbMethod<MixerWithdraw<T>, WebbWithdrawEvents>;
 }
 
 export interface WebbFixedAnchor<T extends WebbApiProvider<any>> {
   // deposit
   deposit: WebbMethod<AnchorDeposit<T, DepositPayload>, MixerDepositEvents>;
   // withdraw
-  withdraw: WebbMethod<AnchorWithdraw<T>, MixerWithdrawEvents>;
+  withdraw: WebbMethod<AnchorWithdraw<T>, WebbWithdrawEvents>;
 }
 
 export interface WebbVariableAnchor<T extends WebbApiProvider<any>> {
   // deposit
   deposit: WebbMethod<VAnchorDeposit<T, DepositPayload>, MixerDepositEvents>;
   // withdraw
-  // withdraw: WebbMethod<VAnchorWithdraw<T>, MixerWithdrawEvents>;
+  // withdraw: WebbMethod<VAnchorWithdraw<T>, WebbWithdrawEvents>;
```

Comment:
Is this intended to be commented out?

Reply:
Yes, this implementation is uncommented and will be implemented in the next PRs